parcel.go

```diff
@@ -1,60 +1,145 @@
+// тут мы работаем с конкретной посылкой
 package main
 
 import (
 	"database/sql"
 )
 
+// Итак, поле db типа *sql.DB - это указатель на соединение с базой данных.
 type ParcelStore struct {
 	db *sql.DB
 }
 
+// Передаем хранилище посылок в качестве аргумента функциям Add() и Get().
 func NewParcelStore(db *sql.DB) ParcelStore {
 	return ParcelStore{db: db}
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
+	stmt, err := s.db.Prepare("INSERT INTO parcel (number, client, status, address, created_at) VALUES (?, ?, ?, ?, ?)")
```

Comment:
id формирует БД, заранее мы его не знаем

Так же можно использовать `s.db.Exec`, чтобы сразу выполнить запрос
```suggestion
	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :created_at)", p.Client, p.Status, p.Address, p.CreatedAt)
```